cab/urls/bookmarks.py

```diff
@@ -1,15 +1,15 @@
-from django.conf.urls import url
+from django.urls import path
 
 from ..views import bookmarks
 
 urlpatterns = [
-    url(r'^$',
-        bookmarks.user_bookmarks,
-        name='cab_user_bookmarks'),
-    url(r'^add/(?P<snippet_id>\d+)/$',
-        bookmarks.add_bookmark,
-        name='cab_bookmark_add'),
-    url(r'^delete/(?P<snippet_id>\d+)/$',
-        bookmarks.delete_bookmark,
-        name='cab_bookmark_delete'),
+    path('',
+         bookmarks.user_bookmarks,
```

Comment:
I'd like to follow Django's guideline of allowing lines long than 79 characters if it helps readability.  I would single line all the path()s

Reply:
I agree with you but I didn't want to change these lines. I changed now